test/unit/modules/node.js

```diff
@@ -313,40 +321,120 @@ describe('node', () => {
-					'should call callback with result containing broadhash = modules.system.getBroadhash result'
-				);
+				it('should return status object with properties', done => {
+					node_module.shared.getStatus(null, (err, status) => {
+						const properties = [
+							'broadhash',
+							'consensus',
+							'currentTime',
+							'secondsSinceEpoch',
+							'height',
+							'loaded',
+							'networkHeight',
+							'syncing',
+						];
+						properties.forEach(property => {
+							expect(status).to.have.property(property);
+						});
+						expect(Object.keys(status).length).to.equal(properties.length);
+						done();
+					});
+				});
 
-				it(
-					'should call callback with result containing consensus = modules.peers.calculateConsensus result'
-				);
+				it('should call callback with result containing broadhash = modules.system.getBroadhash() result', done => {
+					node_module.shared.getStatus(null, (err, status) => {
+						expect(status.broadhash).to.equal(
+							library.modules.system.getBroadhash()
+						);
+						done();
+					});
+				});
 
-				it(
-					'should call callback with result containing height = modules.blocks.lastBlock.get result'
-				);
+				it('should call callback with result containing consensus = modules.peers.getLastConsensus() result', done => {
+					node_module.shared.getStatus(null, (err, status) => {
+						expect(status.consensus).to.equal(
+							library.modules.peers.getLastConsensus()
+						);
+						done();
+					});
+				});
 
-				it(
-					'should call callback with result containing syncing = modules.loader.syncing result'
-				);
+				it('should call callback with result containing height = modules.blocks.lastBlock.get().height result', done => {
+					node_module.shared.getStatus(null, (err, status) => {
+						expect(status.height).to.equal(
+							library.modules.blocks.lastBlock.get().height
+						);
+						done();
+					});
+				});
 
-				it('should call modules.loader.getNetwork');
+				it('should call callback with result containing syncing = modules.loader.syncing() result', done => {
+					node_module.shared.getStatus(null, (err, status) => {
+						expect(status.syncing).to.equal(library.modules.loader.syncing());
+						done();
+					});
+				});
 
-				describe('when modules.loader.getNetwork fails', () => {
-					it(
-						'should call callback with result containing networkHeight = null'
-					);
+				it('should call callback with result containing loaded = modules.loader.loaded() result', done => {
+					node_module.shared.getStatus(null, (err, status) => {
+						expect(status.loaded).to.equal(library.modules.loader.loaded());
+						done();
+					});
 				});
 
-				describe('when modules.loader.getNetwork succeeds and returns network', () => {
-					it(
-						'should call callback with result containing networkHeight = network.height'
-					);
+				describe('modules.loader.getNetwork', () => {
+					beforeEach(done => {
+						stubs.networkHeight = sinonSandbox
```

Comment:
Does this stub get restored after test?

Reply:
@SargeKhan Not at the moment. I just made sure in beforeEach hooks they are being reassigned. Actually i was hoping to get a direction on this. This discussion https://github.com/LiskHQ/lisk/issues/1417 suggests to use a _global_hooks.js file but it doesn't exist yet and I followed the structure in `test/unit/modules/multisignatures.js` since it's written recently by @4miners. 

So please feel free to guide me here.

Reply:
I think it's always better to restore spy/stubs in `afterEach` if they were created in a `beforeEach` block.